src/addons/addonmessage.h

```diff
@@ -38,7 +38,15 @@ class AddonMessage final : public Addon {
       QString formattedDate READ formattedDate NOTIFY retranslationCompleted)
   Q_PROPERTY(Badge badge MEMBER m_badge WRITE setBadge NOTIFY badgeChanged)
 
-  enum Badge { None, Warning, Critical, NewUpdate, WhatsNew, Survey };
+  enum Badge {
+    None,
+    Warning,
+    Critical,
+    NewUpdate,
```

Comment:
super nit... but NewUpdate is redundant.
```suggestion
    Update,
```

Reply:
agree. hate it. but this is not new (linter change) and is already being used by all of the "what's new" addons